### ISSUE TITLE
Changes to metadata ignored when multiple modifications are performed under time limit

### ISSUE THREAD

**Issue by LHLaurini** [cognusion/fuse-copyfs#7]
```
$ mkdir version-dir
$ mkdir mountpoint
$ copyfs-mount $PWD/version-dir mountpoint
/home/user/test/version-dir
$ cd mountpoint/
$ echo content > file; chmod 777 file
$ ls -lh
total 4,0K
-rw-r--r-- 1 user user 8 jan 28 19:37 file
$ chmod 777 file
$ ls -lh
total 4,0K
-rwxrwxrwx 1 user user 8 jan 28 19:37 file
```

As you can see, creating a file and then immediately changing its mode, will cause the mode change to be ignored. This is a problem when running scripts.

The problem seems to be caused by trying to avoid creating unnecessary versions:
https://github.com/cognusion/fuse-copyfs/blob/66b8df2f0b678d20cdff7a42dc4830b90687f981/create.c#L148-L150

If I set `TIME_LIMIT` to 0, the problem disappears.

**Comment by cognusion** [comment]
Yes, this is a known issue. It results in the best experience for 99.9999% of operations but unfortunately does create an eyesore for the odd case.

I will gladly accept a PR that allows dynamic changes to temporarily adjust the value, but am resistant to changes that are either hard-coded or require remounting. 

One solution, if you're looking for brain food, is to externalize TIME_LIMIT so in the event you want to run a script that needs to make microchanges to the file, you could do e.g. `TIME_LIMIT=0 ./my script.sh`.
